conans/test/functional/cmake_test.py

```diff
@@ -262,6 +283,11 @@ def test_clean_sh_path(self):
         cmake.configure(conanfile)
         self.assertNotIn(self.tempdir, conanfile.path)
 
+    @staticmethod
+    def __scape(args):
```

Comment:
avoid private methods (two underscores). They are not used at all in the conan codebase.